source/renderer/app/components/staking/Staking.js

```diff
@@ -1,24 +1,53 @@
 // @flow
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
-import BlockGenerationInfo from './BlockGenerationInfo';
-import StakingSwitch from './StakingSwitch';
-import StakingSystemState from './StakingSystemState';
+import { defineMessages, intlShape } from 'react-intl';
+import DelegationInfo from './DelegationInfo';
 import styles from './Staking.scss';
 
+const messages = defineMessages({
+  heading: {
```

Comment:
@yakovkaravelov we define messages in the component where they are used (they are never passed around). I think these ones belong to: `source/renderer/app/components/staking/DelegationInfo.js`